modules/rhcos-about.adoc

```diff
@@ -122,7 +122,7 @@ The way that Ignition configures machines is similar to how tools like https://c
 
 The Ignition process for an {op-system} machine in an {product-title} cluster involves the following steps:
 
-* The machine gets its Ignition config file. Control plane machines (also known as the master machines) get their Ignition config files from the bootstrap machine, and worker machines get Ignition config files from a master.
+* The machine gets its Ignition config file. Control plane machines get their Ignition config files from the bootstrap machine, and worker machines get Ignition config files from a master.
```

Comment:
FYI I'm seeking input from RHCOS dev on whether we can/should replace "master" with "control plane machine" at the end of this sentence.

Reply:
@mburke5678 I got approvals from the RHCOS team to update the end of the second sentence, so would you mind also making the following change, please?
`s/Ignition config files from a master/Ignition config files from a control plane machine/` 🙏